sdk/react/discoveryPanel.js

```diff
@@ -198,6 +224,40 @@ var DiscoveryPanel = React.createClass({
         </Text>
       </View>);
   },
+
+  renderError: function() {
+    var errorTitleText = "SOMETHING NOT RIGHT! THERE SHOULD BE VIDEOS HERE.";
+    var errorContentText = "(Try Clicking The Discover Button Again On Reload Your Page)";
+    var errorFlexDirectionStyle = {flexDirection: "row"};
+
+    if (this.props.width < widthThreshold) {
+      errorFlexDirectionStyle = {flexDirection: "column"};
+    }
+
+    var errorTitle = Utils.localizedString(this.props.locale, errorTitleText, this.props.localizedString);
+    var errorContent = Utils.localizedString(this.props.locale, errorContentText, this.props.localizedString);
+    var warningIcon = this.props.config.icons.warning.fontString;
+    if (this._isDicoveryError()) {
+      return (
+        <View style={[styles.panelErrorPanel, errorFlexDirectionStyle]}>
+          <View style={styles.panelErrorInfo}>
+            <Text style={styles.panelErrorTitleText}>
+              {errorTitle + "\n"}
+              <Text style={styles.panelErrorContentText}>
```

Comment:
this is fine, but I would have liked if the two <text> dom elements were separate, and not embedded in each other